File: src/Lista01/Questao01_02/negocio/Negocio.java
package Lista01.Questao01_02.negocio;

import Lista01.Questao01_02.dados.Veiculo;
import Lista01.Questao01_02.dados.Cliente;

import java.util.Arrays;

public class Negocio {
    private Veiculo[] veiculos = new Veiculo[2];
    private int numVeiculos = 0;
    private Cliente[] clientes = new Cliente[2];
    private int numClientes = 0;

    public Veiculo[] getVeiculos() {
        return veiculos;
    }

    public void setVeiculo(Veiculo veiculo) {
        this.veiculos[this.numVeiculos++] = veiculo;
    }

    public Cliente[] getClientes() {
        return clientes;
    }

    public void setCliente(Cliente cliente) {
        this.clientes[this.numClientes++] = cliente;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Negocio negocio = (Negocio) o;
        return numVeiculos == negocio.numVeiculos && numClientes == negocio.numClientes && Arrays.equals(veiculos, negocio.veiculos) && Arrays.equals(clientes, negocio.clientes);
    }

    public String toString() {
        return "\n~~~Veículos~~~\n" + Arrays.toString(veiculos) + "\n~~~Clientes~~~\n" + Arrays.toString(clientes);
    }
}

File: src/Lista01/Questao05/Negocio.py
from Veiculo import Veiculo
from Cliente import Cliente

class Negocio:

	def __init__(self):
		self.__veiculos =  []
		self.__numVeiculos = 0
		self.__clientes = []
		self.__numClientes = 0

	def getVeiculos(self):
		return self.__veiculos

	def setVeiculo(self, veiculo : Veiculo):
		self.__veiculos.append(veiculo)

	def getNumVeiculos(self):
		return self.__numVeiculos

	def setNumVeiculos(self):
		self.__numVeiculos = self.__numVeiculos + 1

	def getNumClientes(self):
		return self.__numClientes

	def setNumClientes(self):
		self.__numClientes = self.__numClientes + 1

	def getClientes(self):
		return self.__clientes

	def setCliente(self, cliente : Cliente):
		self.__clientes.append(cliente)

	def __str__(self):
		return 'Veiculos: {}, Clientes: {}'.format(self.__veiculos, self.__clientes)